packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -121,6 +147,31 @@ export class CustomAppComponent implements AfterViewInit {
         alert(`${menuItemText} selected`);
     }
 
+    public onDynamicSelectFilterInput(e: Event): void {
+        const event = e as CustomEvent<SelectFilterInputEventDetail>;
+        const filter = event.detail.filterText;
+        if (filter.length > 0) {
+            window.clearTimeout(this.dynamicSelectFilterTimeout);
+            this.dynamicSelect.loadingVisible = true;
+            this.dynamicSelectFilterTimeout = window.setTimeout(() => {
```

Comment:
When playing with this during the UI WG meeting I found it slightly unexpected that the original items that don't match the filter were still visible while the filter was "processing". Depending on how "real" we want this example to be, we could change that. Or we could decide that overcomplicates things and that the main purpose of the example is for our own testing. I'm fine either way.